src/Tools/ManuallyAlignPointCloud/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
-file(GLOB HEADER_FILES "*.h")
 file(GLOB SOURCE_FILES "*.cpp")
-add_executable(ManuallyAlignPointCloud ${HEADER_FILES} ${SOURCE_FILES})
-target_link_libraries(ManuallyAlignPointCloud Visualization IO Core)
-set_target_properties(ManuallyAlignPointCloud PROPERTIES
-    FOLDER "Tools")
+
```

Comment:
redundant lines?

Reply:
_ManuallyAlignPointCloud_ was treated differently than the other _src/Tools_ projects. Because of that I can't replace the CMake logic for it with a one liner the way I did with the other _src/Tools_ projects.

Reply:
I see. I actually meant to say that let's remove empty lines.